test/grdimage/defcpt.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+# Ensure that grdimage picks up the CPT stored in the netCDF z:cpt attribute
+# https://github.com/GenericMappingTools/gmt/issues/6167
+# Fixed via 
+gmt begin defcpt
+	gmt grdcut @earth_relief_01d_g -R0/10/0/10 -Gtest.nc
+	gmt subplot begin 2x1 -R0/10/0/10 -JQ10c -Fs10c
+		gmt grdimage @earth_relief_01d_g -R0/10/0/10 -B -c
+		gmt grdimage test.nc -B -c
+	gmt subplot end
+gmt end show
```

Comment:
Here we are again creating a new PS. Preferably the test should detect the presence of the cpt but if that's too complicated, at least make the test a tinny figure.

Reply:
I just updated to remove the boundary.  It is a 1010 grid - that is pretty small, no? The size of the PS is 22K, og which 20K is the PSL header.

Reply:
I do not see how I can make a non-plot test to detect that GMT correctly identifies the attribute and stretches the CPT, etc.

Reply:
I like my idea though: It complements the attribute idea since that solution will survive a session whereas the current CPT has a limited scope in where it is found (figure, subplot, inset), etc.